applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h

```diff
@@ -346,10 +347,11 @@ class DistanceModificationProcess : public Process
             {
                 if (rGeometry[itNode].GetSolutionStepValue(DISTANCE)<0.0)
                     inside++;
+                if (rGeometry[itNode].IsFixed(VELOCITY))
```

Comment:
this makes no sense (and i guess it does not compile)

you need to check for the components

       if (rGeometry[itNode].IsFixed(VELOCITY_X))